models/stg_hubspot__owner_team.sql

```diff
@@ -0,0 +1,30 @@
+{{ config(enabled=fivetran_utils.enabled_vars(['hubspot_team_enabled'])) }}
```

Comment:
The tmp model has owner_team and team variables. Can you review and confirm which variables should be used for this stg and tmp combo

Reply:
Thanks--the tmp model is the wrong one. It should only be `hubspot_team_enabled`. Updated to remove!